app/assets/stylesheets/alchemy/sitemap.scss

```diff
@@ -27,7 +27,35 @@
     left: 50% !important;
   }
 }
-
+#sitemap li {
+  overflow: visible;
+}
+.invalid_user_type_link {
```

Comment:
Please use the new hint css class from https://github.com/AlchemyCMS/alchemy_cms/pull/1164